src/otx/core/utils/config.py

```diff
@@ -43,12 +43,13 @@ def to_list(dict_: dict) -> dict:
 
 
 def convert_conf_to_mmconfig_dict(
-    cfg: DictConfig,
+    cfg: DictConfig | dict,
```

Comment:
Changing this will unnecessarily increase the number of file changes, which can be frustrating for reviewers, so I would suggest fixing this in one go in the near future.